[ignition-msgs-5] compile error DSO missing from command line

While rebuilding gazebo classic packages I get the following error:
```bash
/sbin/ld: CMakeFiles/ign_msgs_gen.dir/Generator.cc.o: undefined reference to symbol '_ZN4absl12lts_2023012513hash_internal15MixingHashState28CombineLargeContiguousImpl64EmPKhm'
/sbin/ld: /usr/lib/libabsl_hash.so.2301.0.0: error adding symbols: DSO missing from command line
```
This error occurs while building _ignition-mgs-5_ and _gazebo_

--- a/ignition-msgs-5/.SRCINFO
+++ b/ignition-msgs-5/.SRCINFO
@@ -1,7 +1,7 @@
 pkgbase = ignition-msgs-5
 	pkgdesc = Messages for Gazebo robot simulation.
 	pkgver = 5.11.0
-	pkgrel = 4
+	pkgrel = 5
 	url = https://gazebosim.org/libs/msgs
 	arch = i686
 	arch = x86_64

--- a/ignition-msgs-5/PKGBUILD
+++ b/ignition-msgs-5/PKGBUILD
@@ -22,6 +22,7 @@ build() {
   cd "${srcdir}/${_dir}"
   mkdir -p build && cd build
 
+  export LDFLAGS="-Wl,--copy-dt-needed-entries"
   cmake .. -DCMAKE_BUILD_TYPE="Release" \
            -DCMAKE_INSTALL_PREFIX="/usr" \
            -DCMAKE_INSTALL_LIBDIR="lib" \

--- a/ignition-msgs-5/PKGBUILD
+++ b/ignition-msgs-5/PKGBUILD
@@ -1,9 +1,10 @@
 # Maintainer: acxz <akashpatel2008 at yahoo dot com>
 # Contributor: Homalozoa <nx dot tardis at gmail dot com>
+# Contributor: Bernd Müller <github@muellerbernd.de>
 
 pkgname=ignition-msgs-5
 pkgver=5.11.0
-pkgrel=4
+pkgrel=5
 pkgdesc="Messages for Gazebo robot simulation."
 arch=('i686' 'x86_64')
 url="https://gazebosim.org/libs/msgs"

